EventFilter/L1TRawToDigi/plugins/L1TValidationEventFilter.cc

```diff
@@ -95,22 +91,15 @@ bool
 L1TValidationEventFilter::filter(edm::Event& iEvent, const edm::EventSetup& iSetup)
 {
   using namespace edm;
-  
-  edm::Handle<FEDRawDataCollection> feds;
-  iEvent.getByToken(fedData_, feds);
 
-  if (!feds.isValid()) {
-    LogError("L1T") << "Cannot unpack: no FEDRawDataCollection found";
+  Handle<int> triggerCount;
+  iEvent.getByLabel(InputTag(src_.label(),"triggerCount"), triggerCount);
```

Comment:
In principle it can all be configurable, but the intention was to only have the label configurable to match the _unpacker_  module label used. In this case the product <int> of instance label "triggerCount" is of interest.  This instance label is assigned to the product by the TcdsRawToDigi _unpacker_ and it differentiates it from other <int> product put in Event.